server/src/interfaces/router.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from "express";
+
+export interface IRouter extends Router {
```

Comment:
Export class -> this can allow us to do in the future new IRouter(); if we'll need it